extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -134,7 +134,9 @@ export const popupMessageListener = (request: Request) => {
     });
   };
 
-  const _fundAccount = async (publicKey: string) => {
+  const fundAccount = async () => {
+    const { publicKey } = request;
```

Comment:
instead of automatically funding with friendbot on Testnet, a user has to manually trigger this from the UI now